### PR TITLE
[SPMM-9186] - bump operator bundle version

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/main/README.md#contributing-to-the-kogito-operator)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains a link to the JIRA issue
- [ ] Pull Request contains a description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

**Review by desmax74 — APPROVED** [review]

**Review by antncl — APPROVED** [review]
LGTM

File: config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -89,4 +88,5 @@ spec:
   maturity: beta
   provider:
     name: Red Hat
+  replaces: rhpam-kogito-operator.v7.12.0-1
```
Comment (jakubschwan) [inline config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml:91]: Replace version not exists, so this csv will fail build of indexing image. There should be used `7.12.0-1` see https://github.com/kiegroup/rhpam-kogito-operator/blob/7.12.0.GA/bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml#L421-L422

Comment (spolti) [reply config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml:91]: without v?

Comment (jakubschwan) [reply config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml:91]: without this part `rhpam-kogito-operator.v` as previous version is 7.12.0-1

Comment (jakubschwan) [reply config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml:91]: or without a v, we have it like that, without the v in the kie cloud operator

**Pull request merged**